.github/workflows/ci.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: [ '3.0', '2.7', '2.6', '2.5', 'jruby', 'truffleruby', 'truffleruby+graalvm' ]
+        ruby: [ '3.0', '2.7', '2.6', '2.5', 'jruby', 'truffleruby', 'truffleruby+graalvm', 'truffleruby+graalvm-head' ]
```

Comment:
Yeah, the test suite is flaky enough, I'd rather not add truffle-head (not saying it's flay by itself, but it's one more thing that can go wrong.

Reply:
OK, let's wait until the 22.2 release.
At least running it in CI revealed there was another issue, I pushed a fix for that.